apps/vaporgui/VPushButtonWithDoubleClick.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+class VPushButtonWithDoubleClick : public QPushButton {
+    Q_OBJECT
+    using QPushButton::QPushButton;
+    void mouseDoubleClickEvent(QMouseEvent * e) {
+        emit doubleClicked();
+    }
```

Comment:
This will get triggered if you double click on the container, not on the button.

Reply:
This inherits from QPushButton, not VPushButton